fuel/utils/exc.py

```diff
@@ -0,0 +1,86 @@
+"""Exceptions used by basic support utilities."""
+__author__ = "Ian Goodfellow"
```

Comment:
We don't usually put the author. Can you just acknowledge Ian and pylearn in the docstring?